action.yml

```diff
@@ -11,6 +11,8 @@ runs:
     - uses: actions/setup-go@v3
       with:
         go-version-file: '${{ github.action_path }}/go.mod'
+      env:
+        CGO_ENABLED: 0
```

Comment:
Should we instead add this to https://github.com/docker-library/bashbrew/blob/6334a4f8e8b210d771d5ba55755f321e57a4ca08/bashbrew.sh#L9 ?

(with `: "${CGO_ENABLED:=0}"`)

Reply:
Yeah I think that makes sense.
```console
$ git grep -i cgo
Dockerfile:RUN CGO_ENABLED=0 ./bashbrew.sh --version; \
Dockerfile.release:ENV CGO_ENABLED 0
action.yml:        CGO_ENABLED: 0
```

Reply:
Ok, now moved to `bashbrew.sh`. We could drop it from the `Dockerfile` but it seems fine to be explicit there and in the `Dockerfile.release`.
```console
$ git grep -i cgo
Dockerfile:RUN CGO_ENABLED=0 ./bashbrew.sh --version; \
Dockerfile.release:ENV CGO_ENABLED 0
bashbrew.sh:: "${CGO_ENABLED:=0}"
bashbrew.sh:export GO111MODULE=on CGO_ENABLED
```